app/src/main/res/values/strings.xml

```diff
@@ -94,6 +94,8 @@
     <string name="account" translatable="false">Account</string>
     <string name="logout">Log out</string>
     <string name="logout_message">Are you sure you want to log out?</string>
+    <string name="employee_mode">TUM Employee View Mode</string>
```

Comment:
Maybe that’s just me, but I find that this sounds a little odd. Maybe @kordianbruck or @pfent have an idea?

Reply:
> Employee Mode 

would really be enough.

Reply:
I'd personally say "Non-Student Mode" :man_shrugging:

Reply:
I like Employee Mode, it's short. What about the german label?

Reply:
"Mitarbeitermodus"?